Core/src/Geometry/TrackingGeometry.cpp

```diff
@@ -16,13 +16,15 @@
 Acts::TrackingGeometry::TrackingGeometry(
     const MutableTrackingVolumePtr& highestVolume,
     const IMaterialDecorator* materialDecorator,
-    const GeometryIdentifierHook& hook)
+    const GeometryIdentifierHook& hook, const Logger& logger)
     : m_world(highestVolume),
       m_beam(Surface::makeShared<PerigeeSurface>(Vector3::Zero())) {
+  std::unique_ptr<const Logger> mylogger = logger.clone("TrackingGeometry");
```

Comment:
This works, but since `logger` is never used for anything else, why not simply pass through directly (i.e. why the `clone`)?

Reply:
On this. It is used here to rename "source". Maybe bad idea. If I could just clone it w/o capturing I think we would leak memory. No?

Reply:
Indeed. Given that the logger can have any name I thought that this way we can make the geometry printout kind of stable. But if this is no tin line of logging policy, surely can change.

Reply:
Removed the loden, just passing down the log now.